src/FlightDisplay/GuidedValueSlider.qml

```diff
@@ -239,6 +261,41 @@ Item {
                                         QGroundControl.unitsConversion.appSettingsSpeedUnitsString : 
                                             QGroundControl.unitsConversion.appSettingsVerticalDistanceUnitsString
         }
+
+        QGCMouseArea {
+            anchors.fill: parent
+            onClicked: {
+                sliderValueTextField.text = _clampedSliderValue(_sliderValue)
+                sliderValueTextField.visible = true
+                sliderValueTextField.forceActiveFocus()
+            }
+        }
+
+        QGCTextField {
+            id:                 sliderValueTextField
+            anchors.leftMargin: indicatorCanvas.pointerWidth
+            anchors.fill:       parent
+            showUnits:          true
+            unitsLabel:         valueLabel.unitsString
+            visible:            false
+            validator:          RegularExpressionValidator { regularExpression: /^-?(\d{1,3})([.]\d{1})?$/ }
+
+            onEditingFinished: {
+                visible = false
+                focus = false
+                setCurrentValue(_clampedSliderValue(parseFloat(text)))
+            }
+
+            Connections {
+                target: control
+                on_sliderValueChanged: sliderValueTextField.visible = false
+            }
+
+            Connections {
```

Comment:
I didn't plan to pull this one across since it didn't seem to be needed. I think hiding on slide is better.